docs/source/index.rst

```diff
@@ -1,6 +1,10 @@
+====
 Dask
 ====
 
+Dask  is a simple task scheduling system that breaks up large computations into many small ones.  
+The word "dask" comes from a combination of "DAG" (for directed acyclic graph) and "task." 
```

Comment:
The word dask has many possible interpretations.  It has no particular meaning.  This is somewhat by intent.